TShirtSizeCalculator.cpp

```diff
@@ -3,17 +3,23 @@
 #include "TShirtSizeCalculator.h"
 
 char Size(int cms) {
-    const int SMALL_MAX = 37;
-    const int MEDIUM_MAX = 41;
+    const int SMALL_MAX = 38;

```

Comment:
Good naming. Further suggestion: include the unit in the name. How does this sound: `MAX_SHOULDER_CMS_FOR_SMALL`
Then there will be lower possibility of a cms-inches confusion